### PR TITLE
Modify api to support edition of closed entity

### PR DESCRIPTION
**Pull request by Marx314**
Modify api to support edition of closed entity
docker-compose for integration testing

### CODE REVIEW

**Comment by fguillot** [comment]
We should have a different docker-compose.yaml file to run integration tests. The default one should be used in real world environment, not to run tests.

File: almanach/core/controller.py [hunk 1]
```diff
@@ -107,6 +107,12 @@ def rebuild_instance(self, instance_id, distro, version, os_type, rebuild_date):
             instance.last_event = rebuild_date
             self.database_adapter.insert_entity(instance)
 
+    def update_inactive_entity(self, instance_id, start, end, **kwargs):
+        inactive_instances = self.database_adapter.list_entities_by_id(instance_id, start, end)
+        instance = inactive_instances[0]
```
Comment (fguillot) [inline almanach/core/controller.py]: What happens if `inactive_instances` is empty?

Comment (Marx314) [inline almanach/core/controller.py]: it blow, it's iterative work

**Comment by fguillot** [comment]
👍

**Comment by pmillette** [comment]
👍

**Comment by maxc0c0s** [comment]
👍

**Pull request merged**